pkgs/development/python-modules/drawilleplot/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, drawille
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "drawilleplot";
+  version = "0.1.0";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ZEDroo7KkI2VxdESb2QDX+dPY4UahuuK9L0EddrxJjQ=";
+  };
+
+  propagatedBuildInputs = [
+    drawille
+    matplotlib
+  ];
+
+  pythonImportsCheck = [ "drawilleplot" ];
```

```suggestion
  pythonImportsCheck = [
    "drawilleplot"
   ];
```

One item per line as before. Also for the other packages.